nucliadb/nucliadb/search/predict.py

```diff
@@ -289,9 +313,12 @@ async def check_response(
             data = await resp.json()
             raise LimitsExceededError(402, data["detail"])
 
-        error = (await resp.read()).decode()
-        logger.error(f"Predict API error at {resp.url}: {error}")
-        raise SendToPredictError(f"{resp.status}: {error}")
+        try:
+            detail = await resp.json()
+        except Exception:
```

Comment:
I'd be better to handle a more specific exception

Reply:
agree! changed